app.rambox.ramboxce.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.rambox.ramboxce</id>
+  <metadata_license>GPL</metadata_license>
+  <name>Rambox Community Edition</name>
+  <developer_name>Rambox</developer_name>
+  <url type="homepage">https://rambox.app/#home</url>
+  <summary>Put all your online Services in one Application!</summary>
+  <description>
+    <p>Rambox gives you the possibility to add common services many times you need, all in one place.</p>
+		<p>It's perfect for people who work with many services for business and private accounts. (inofficial Flatpak)</p>
```

Comment:
The appdata should be generally improved, both description and number of screenshots.